app/src/main/res/values/strings.xml

```diff
@@ -730,23 +730,23 @@
     <string name="show_thumbnail_summary">Use thumbnail for both lock screen background and notifications</string>
     <string name="recent">Recent</string>
     <string name="chapters">Chapters</string>
-    <string name="no_app_to_open_intent">No app on your device can open this</string>
+    <string name="no_app_to_open_intent">Install an app that can open the file first</string>
```

Comment:
Please don't change this string, it is not only used for files.

Reply:
```suggestion
    <string name="no_app_to_open_intent">Install an app that can open this first</string>
```